src/libraries/System.Private.CoreLib/src/System/Numerics/IDecimalFloatingPointIeee754.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Numerics
+{
+    /// <summary>Defines an IEEE 754 floating-point type that is represented in a base-10 format.</summary>
+    /// <typeparam name="TSelf">The type that implements the interface.</typeparam>
+    public interface IDecimalFloatingPointIeee754<TSelf>
```

Comment:
Probably want this as `internal` until we can get everything in.

I think we'd also want it in `System.Runtime.Numerics` alongside the main types.